src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/RouteRequestTransitDataProviderFilter.java

```diff
@@ -103,6 +109,12 @@ public boolean tripTimesPredicate(TripTimes tripTimes, boolean withFilters) {
       }
     }
 
+    if (requireCarsAllowed) {
+      if (trip.getCarsAllowed() != CarAccess.ALLOWED) {
```

Comment:
Nitpick but this could be a single if

Reply:
I changed this for both requireBikesAllowed and requireCarsAllowed